examples/running/la-grande-classique-paris-versailles.gpx

```diff
@@ -0,0 +1,189 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
```

Comment:
@JulesL2 

I skipped two parts of your initial PR to focus on correctly using the UiManager. I let you add them back in a new PR
- _Layout classvar in BaseVerticalLayout
- Two Lines Stats